libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -1,9 +1,12 @@
 #include "AP_Camera_Backend.h"
+#include <AP_Mount/AP_Mount.h>
+#include <AP_Terrain/AP_Terrain.h>
 
 #if AP_CAMERA_ENABLED
 #include <GCS_MAVLink/GCS.h>
 #include <AP_GPS/AP_GPS.h>
-#include <AP_Mount/AP_Mount.h>
+#pragma GCC diagnostic push
+#pragma GCC diagnostic error "-Wframe-larger-than=2400"
```

Comment:
This doesn't look good.

There's a reason this warning is enabled and set at the current level; stack-overflows kill vehicles.

Why was this required?  Can we fix it in some other way?

Reply:
See this discussion [here](https://discuss.ardupilot.org/t/frame-larger-than-compilation-error-with-debug-option/66754), I got the same problem. Maybe it is fixed in the new version.

Reply:
its fixed in the new patch